setup.py

```diff
@@ -4,7 +4,7 @@
 import sys
 from setuptools import setup, find_packages
 
-install_requires=['catkin_pkg >= 0.1.28', 'rosdistro >= 0.5.0', 'rospkg', 'PyYAML', 'setuptools']
+install_requires=['catkin_pkg >= 0.1.28', 'rosdistro >= 0.6.7', 'rospkg', 'PyYAML', 'setuptools']
```

Comment:
Do you want to bump the minimum version to 0.6.8 (same below)?

Reply:
Done.